src/pages/discovery/discovery.scss

```diff
@@ -23,18 +24,18 @@
   }
 
   #filtersContainer {
-    width: 27vw;
+    width: 80vw;
+    margin: 0 auto;
```

Comment:
This margin is immediately overwritten. Review to decide which one you intended to keep.